### PR TITLE
test: Update test/common, mostly for animation waiting

### PR DESCRIPTION
**Pull request by mvollmer**
Changes:

- Now assert_pixels waits for animations to be over before taking the
  screenshot.

- assert_pixels also removes focus before screenshotting, so we need
  to adjust some references.

- We can decide locally which image does the pixel tests and changes
  to TEST_OS_DEFAULT wont affect that.

### CODE REVIEW

**Comment by martinpitt** [comment]
The changes look good, especially the testSerialConsole reference image update -- all of a sudden this makes sense!

Needs ref image updating.

**Comment by mvollmer** [comment]
We can stop ignoring the triangles now, I think.

**Comment by mvollmer** [comment]
> The changes look good, especially the testSerialConsole reference image update -- all of a sudden this makes sense!

Hmm, is the text in the console stable enough?  Let's see!

**Comment by martinpitt** [comment]
> is the text in the console stable enough?

I am convinced it is not. At least the generated host key will differ between each run, The IP addresses also sound brittle. We may need to ignore the console contents.

File: test/check-machines-create [hunk 1]
```diff
@@ -1060,8 +1060,6 @@ class TestMachinesCreate(VirtualMachinesCase):
             return self
 
         def assert_pixels(self, subtag=None):
-            # focus away from any input field, to avoid the text cursor in screenshots
-            self.browser.focus("#create-vm-dialog .btn-cancel")
```
Comment (martinpitt) [inline test/check-machines-create:1064]: That is correct, but will again change the pixel tests. The update commit was earlier than this change, but let's see if you included it there already.

Comment (mvollmer) [reply test/check-machines-create:1064]: No changes were necessary. :-)

**Comment by mvollmer** [comment]
> Yup, both IP addresses and uptime change, see [failed pixel test](https://logs.cockpit-project.org/logs/pull-481-20211214-131722-3c08113d-fedora-35-firefox/pixeldiff.html#TestMachinesConsoles-testSerialConsole-vm-details-console-serial)

Ignored.  

I was surprised that the serialConsole test acutally shows the VNC console. Is that expected?

**Comment by mvollmer** [comment]
> I was surprised that the serialConsole test acutally shows the VNC console. Is that expected?

Clicking the "Expand" button doesn't keep the selection from the dropdown, but seems to go back to some default, maybe.

**Comment by martinpitt** [comment]
> I was surprised that the serialConsole test acutally shows the VNC console. Is that expected?

Good point -- no, that's surely a bug.

**Review by martinpitt — APPROVED** [review]
Thank you! packit tests are doomed until PR #484, but they don't do pixel tests. So I think it's safe to land this.

**Pull request merged**